app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,31 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if isinstance(lhs, str):
+            return int(lhs) + rhs
+        return lhs + rhs
+
+    @staticmethod
+    def format_ordinal(number_to_format, determiner=None):
+        number_to_format = str(number_to_format)
+        if number_to_format.endswith(("11", "12", "13")):
+            number_to_format += "th"
+        elif number_to_format.endswith("1"):
+            number_to_format += "st"
+        elif number_to_format.endswith("2"):
+            number_to_format += "nd"
+        elif number_to_format.endswith("3"):
+            number_to_format += "rd"
+        else:
+            number_to_format += "th"
+        if determiner == "a_or_an":
+            if number_to_format.startswith("8") or number_to_format == "18th":
+                return "an " + str(number_to_format)
+            return "a " + str(number_to_format)
```

Comment:
`number_to_format` is already a string. It would be better to assign it to a var instead of mutating it. We should use f-Strings here.

This can be starts with '8' or starts with '18'

Reply:
Everything done except startswith() for "8" but not for "18", not everything that starts with "18" will get an "an"😂